web/concrete/src/Config/Repository/Repository.php

```diff
@@ -56,6 +57,11 @@ public function save($key, $value)
         if ($this->saver->save($item, $value, $this->environment, $group, $namespace)) {
             $this->load($group, $namespace, $this->getCollection($group, $namespace));
 
+            switch ($key) {
+                case 'concrete.seo.url_rewriting':
+                    Core::make('helper/file/htaccess')->setCurrentRewriteRules((bool) $value);
+                    break;
+            }
```

Comment:
:-1: we do not want to couple Repository to anything, these config values are already stored in memory, just access them from the repository when you need them.